### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by bnelson7**

### CODE REVIEW

File: frontend/src/components/comments/comment_index_item.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const CommentIndexItem = props => {
+    return (
+        <div>
+            {comment}
```
Comment (haydenlinder) [inline frontend/src/components/comments/comment_index_item.jsx]: comment is undefined

File: routes/api/comments.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ router.post("/",
 
 router.get("/tickets/:ticketId", (req, res) => {
     Comment
-        .find({ ticket: req.params.ticked})
+        .find({ ticket: req.params.ticket})
```
Comment (haydenlinder) [inline routes/api/comments.js]: ticketId

File: frontend/src/components/comments/comment_index_item.jsx (hunk 1 above)
Comment (bnelson7) [reply frontend/src/components/comments/comment_index_item.jsx]: yeah thats still work in progress

File: frontend/src/components/comments/comment_index_item.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const CommentIndexItem = props => {
+    return (
+        <div>
+            {props.ticket.comments}
```
Comment (haydenlinder) [inline frontend/src/components/comments/comment_index_item.jsx]: props.comment

**Pull request merged**